static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -150,37 +166,59 @@ export function EventFeatureFlagList({
     [openDrawer, event, group, project, hydratedFlags, organization, sortBy, orderBy]
   );
 
-  if (!hydratedFlags.length) {
+  useEffect(() => {
+    if (hasFlags) {
+      trackAnalytics('flags.table_rendered', {
+        organization,
+        numFlags: hydratedFlags.length,
+      });
+    }
+  }, [hasFlags, hydratedFlags.length, organization]);
+
+  // TODO: for LD users, show a CTA in this section instead
+  // if contexts.flags is not set, hide the section
+  if (!hasFlagContext) {
     return null;
   }
 
   const actions = (
     <ButtonBar gap={1}>
       {feedbackButton}
-      <Button
-        size="xs"
-        aria-label={t('View All')}
-        ref={viewAllButtonRef}
-        title={t('View All Flags')}
-        onClick={() => {
-          isDrawerOpen ? closeDrawer() : onViewAllFlags();
-        }}
-      >
-        {t('View All')}
-      </Button>
-      <Button
-        aria-label={t('Open Feature Flag Search')}
-        icon={<IconSearch size="xs" />}
-        size="xs"
-        title={t('Open Search')}
-        onClick={() => onViewAllFlags(FlagControlOptions.SEARCH)}
-      />
-      <FeatureFlagSort
-        orderBy={orderBy}
-        sortBy={sortBy}
-        setSortBy={setSortBy}
-        setOrderBy={setOrderBy}
-      />
+      {hasFlagContext && (
```

Comment:
Do we still need this after the early return check above?
```suggestion
      {(
```

Reply:
keeping this for now since it'll be useful when we add in the LD feature flag